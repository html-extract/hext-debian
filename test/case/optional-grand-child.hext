<div @text:no_span>
  <p>
    <?span />
  </p>
</div>
