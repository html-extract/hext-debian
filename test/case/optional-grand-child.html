<html>
  <head></head>
  <body>
  <div>
    <img src='src' />
    <p>no <b>span</b></p>
  </div>
  </body>
</html>
