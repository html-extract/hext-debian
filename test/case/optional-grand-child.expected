{"no_span":"no span"}
